lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -219,6 +219,7 @@ public Task createTask(Task taskToCreate)
                 validateObjectReference(task.getPrimaryObjRef(), "primary ObjectReference", "Task");
                 PrioDurationHolder prioDurationFromAttachments = handleAttachments(task);
                 standardSettings(task, classification, prioDurationFromAttachments);
+                handleCallbackState(task);
```

Comment:
why don't we simply set it to NONE? It is always a new task, isn't it?

Reply:
Yes, it is a new task. But if the callback_info map contains the key "callbackState", we use the associated value to initialize the callback_state database field. That is the way how the adapter creates taskana tasks with callback_state = CALLBACK_PROCESSING_REQUIRED